### ISSUE TITLE
Implement anGPT/nGPT in torchtitan

### ISSUE THREAD

**Issue by JoergFranke** [OpenEuroLLM/Taskboard#100]
Implement anGPT in TorchTitan to test a few extensions and rework the scaling law we derived in https://arxiv.org/abs/2505.22014

The goal is to provide an architecture for LLM pretraining that does not require learning rate warmup and regularization but increases convergence speed up to 40% while allowing larger batch sizes and therefore more efficient training. The scaling trend for the learning rate selection may be more stable.

**Comment by JoergFranke** [comment]
Its implemented and we get the same performance as in the https://arxiv.org/abs/2505.22014 paper
